tests/helpers/test_script.py

```diff
@@ -141,6 +141,38 @@ def record_event(event):
         assert not script_obj.is_running
         assert len(events) == 2
 
+    def test_delay_template(self):
+        """Test the delay as a template."""
+        event = 'test_evnt'
+        events = []
+
+        def record_event(event):
+            """Add recorded event to set."""
+            events.append(event)
+
+        self.hass.bus.listen(event, record_event)
+
+        script_obj = script.Script(self.hass, [
+            {'event': event},
+            {'delay': '00:00:{{ 5 }}'},
```

Comment:
Ah, well it's the same unit test as the untemplated one,  want me to adjust the existing test as well?